package.json

```diff
@@ -70,7 +71,7 @@
     "education",
     "programming"
   ],
-  "license": "MIT",
+  "license": "MPL-2.0",
```

Comment:
The code itself should remain MIT. Please do a better job at disclosing this type of change in the future

Reply:
Ah OK I thought it was slightly out of date seeing as the LICENSE file was different. Sorry.

Reply:
Oh wait, I apologize deeply. I thought we were using MIT and not MPL. Let's revert this change from this PR, make a new one. Even so, changes such as this could be broadly sweeping in implication in the wrong context (not this one) so pointing this type of erranious value as it's own issue would be best in the future. 

Apologies once again for the confusion

Reply:
Redirected to #115